ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -151,5 +159,22 @@ void RSkInputEventManager::sendNotificationWithEventType(std::string eventType,
 }
 #endif //TARGET_OS_TV
 
+int RSkInputEventManager::registerAddListener(inputEventClientCallback clientCallback){
+  RNS_LOG_DEBUG("[registerAddListener] ");
+  eventCallbackMutex_.lock();
+  callbackId_++;
+  eventCallbackMap_.insert({callbackId_,clientCallback});
+  eventCallbackMutex_.unlock();
+  return callbackId_;
```

Comment:
Better to use scoped event lock. There could be chance that two threads calling this function but return value can be same since it is outside the lock